plugin/storage/grpc/shared/plugin_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package shared
+
+import (
+	"context"
+	"testing"
+
+	"github.com/jaegertracing/jaeger/storage/dependencystore"
+	dependencyStoreMocks "github.com/jaegertracing/jaeger/storage/dependencystore/mocks"
+	"github.com/jaegertracing/jaeger/storage/spanstore"
+	spanStoreMocks "github.com/jaegertracing/jaeger/storage/spanstore/mocks"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
```

Comment:
these are external imports and should be placed in a group above the internal imports

Reply:
`make fmt` does this for you

Reply:
thanks